### PR TITLE
Exploration Expansion: First POI

### PR DESCRIPTION
**Pull request by SleepingZi**
This is a test merge for a new POI to make sure I'm doing this correctly.

This adds a new POI to V4, the Speakeasy, a place to chill from the long day of exploring.
unknown

### CODE REVIEW

**Comment by ShudderEve** [comment]
What about the existing beach bar + kitchen + shore area? Are those not enough? I'd have thought that V4 PoIs would be more exploration and puzzles than more bars.

**Comment by dwinters99** [comment]
>What about the existing beach bar + kitchen + shore area? Are those not enough?

I would say that more places to hang out aren't bad at all. 

I would also say, in my opinion, that it would probably be a nice to have this PoI be in a partial state of disrepair. People have responded very well in the past to things needing construction such as the debris field ships. Also, this does seem a tad blocky, but that's an easy issue to fix.

**Comment by SleepingZi** [comment]
> I would also say, in my opinion, that it would probably be a nice to have this PoI be in a partial state of disrepair. People have responded very well in the past to things needing construction such as the debris field ships. Also, this does seem a tad blocky, but that's an easy issue to fix.
I see what you mean here, however I like the finished state a little more, it gives the 'still in use' vibe that I'm trying to go for. I want this to be something that gives the caves some notion that they're still in use for various purposes, and not just a place of abandoned structures and hostile creatures.
I do see what you mean by it being blocky, and I have to agree, though I don't see a way to fix that without making the POI larger.

**Comment by SleepingZi** [comment]
Revision 1.1: Made structure more 'natural' and added a jukebox to get the 2220's vibe

File: maps/submaps/surface_submaps/mountains/mountains_areas.dm [hunk 1]
```diff
@@ -163,3 +163,7 @@
 /area/submap/spatial_anomaly
 	name = "POI - Spatial Anomaly"
 	ambience = AMBIENCE_FOREBODING
+
+/area/submap/Speakeasy
```
Comment (Novacat) [inline maps/submaps/surface_submaps/mountains/mountains_areas.dm:167]: //VOREStation Add

File: maps/submaps/surface_submaps/mountains/mountains.dm [hunk 2]
```diff
@@ -363,3 +364,11 @@
 	mappath = 'maps/submaps/surface_submaps/mountains/spatial_anomaly.dmm'
 	cost = 20
 	fixed_orientation = TRUE
+
+/datum/map_template/surface/mountains/normal/Speakeasy //VOREStation Edit
```
Comment (Novacat) [inline maps/submaps/surface_submaps/mountains/mountains.dm:368]: This should probably be changed to //VOREStation Add

**Review by Novacat — CHANGES_REQUESTED** [review]

File: maps/submaps/surface_submaps/mountains/mountains.dm [hunk 3]
```diff
@@ -41,6 +41,8 @@
 #include "Cliff1.dmm"
 #include "excavation1.dmm"
 #include "spatial_anomaly.dmm"
+#include "speakeasy_vr.dmm"
+#include "Campsite_vr.dmm"
```
Comment (Novacat) [inline maps/submaps/surface_submaps/mountains/mountains.dm:45]: I think you added this by accident?

**Review by Novacat — CHANGES_REQUESTED** [review]

File: maps/submaps/surface_submaps/mountains/mountains.dm (hunk 3 above)
Comment (SleepingZi) [reply maps/submaps/surface_submaps/mountains/mountains.dm]: yes, that was a result of me using upload to make a pull, I've fixed the issue.

**Review by Novacat — APPROVED** [review]

**Pull request merged**